index.js

```diff
@@ -221,6 +221,12 @@ function grype_render_rules(vulnerabilities) {
         let ruleID = "ANCHOREVULN_"+v.vulnerability.id+"_"+v.artifact.type+"_"+v.artifact.name+"_"+v.artifact.version;
         if (!ruleIDs.includes(ruleID)) {
           ruleIDs.push(ruleID);
+          // Entirely possible to not have any links whatsoever
+          var link = v.vulnerability.id;
```

Comment:
Not blocking, but in general `var` should never be used, and it's safe to use `let` in its place

Reply:
I slipped on this one and have updated it to fix it. I ran tests locally against the reproducer, and it passed